fuzzing/libfuzzer/common/config/system_packages.yaml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
Prashanth you don't need so many packages to just build, install and run Gramine. follow this link: https://gramine.readthedocs.io/en/stable/devel/building.html#common-dependencies

and this libfuzzer we won't run on all distros, assume its Ubuntu22.04.

Reply:
Updated